JunitHomework/src/main/java/string/utility/SpaceRemover.java

```diff
@@ -0,0 +1,26 @@
+package string.utility;
+
+public class SpaceRemover {
+    public String before;
```

Comment:
String before has not a meaningful naming. 
Maybe String textInput; looks better